Description: Fix chinese locale code
 Change locale code from cz to zh_CN.
 The correct way to do this is to rename the directory. It's done
 this way to avoid a massive quilt patch.
Author: James Cowgill <james410@cowgill.org.uk>
Applied-Upstream: https://github.com/eranif/codelite/commit/be5168e5b288ef25efcd21a17b73fcadc3f0ecd2
Last-Update: 2014-04-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/LiteEditor/CMakeLists.txt
+++ b/LiteEditor/CMakeLists.txt
@@ -135,11 +135,16 @@
 )
 
 install(
-  DIRECTORY ${CL_SRC_ROOT}/translations/
-  DESTINATION ${CL_PREFIX}/share/locale
-  USE_SOURCE_PERMISSIONS
+  DIRECTORY ${CL_SRC_ROOT}/translations/cs/
+  DESTINATION ${CL_PREFIX}/share/locale/cs
   FILES_MATCHING PATTERN "codelite.mo"
-) 
+)
+
+install(
+  DIRECTORY ${CL_SRC_ROOT}/translations/cz/
+  DESTINATION ${CL_PREFIX}/share/locale/zh_CN
+  FILES_MATCHING PATTERN "codelite.mo"
+)
 
 if (UNIX AND CL_COPY_WX_LIBS) # If we're packaging with wx2.9
     # We need to deduce the location of the wx libs to be installed
